explainaboard/metrics/qa_table_text_hybrid.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import annotations
+
+import abc
+from dataclasses import dataclass
+from typing import Optional
+
+import numpy as np
+
+from explainaboard.metrics.metric import (
+    Metric,
+    MetricConfig,
+    MetricStats,
+    SimpleMetricStats,
+)
+from explainaboard.metrics.registry import register_metric_config
+from explainaboard.utils.eval_utils import qa_table_text_hybrid as eval_util
```

Comment:
I know we discussed this face-to-face, but I guess I still don't really understand the motivation for splitting these functions into a separate class. It seems that they could be made private functions within this file?

Reply:
>"It seems that they could be made private functions within this file?"

The motivations are:
* I intentionally make evaluation scripts in `explainaboard/metric` shorter, which allows users easy to follow.
* there are a lot of preprocessing functions for this task (see [this one](https://github.com/NExTplusplus/L2I/blob/main/evaluate.py)), which are not docstring-ed in the original code. In the future, we will potentially encounter similar situations. Such code could be put in the folder of `eval_util`.

Reply:
OK, how about as a halfway solution we move `explainaboard/utils/eval_utils/qa_table_text_hybrid.py` to `explainaboard/metrics/auxiliary/qa_table_text_hybrid_auxiliary.py` or something. That would make the metrics directory self-contained but still have this separation.

Reply:
looks nice